src/Assignment1/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+
+namespace Assignment1
+{
+    /// <summary>
+    /// This is main class <see cref="Program"/>
+    /// </summary>
+    public class Program
+    {/// <summary>
+     /// This is main function
+     /// </summary>
+     /// <param name="args">This is a sample argument</param>
+        public static void Main(string[] args)
+        {
+            Console.Write("Enter input1 : ");
+            int input1 = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Enter input2 : ");
+            int input2 = Convert.ToInt32(Console.ReadLine());
+            MathUtils obj = new MathUtils();
+
+            Console.WriteLine($"Addition of given inputs : {obj.Add(input1, input2)}");
+            Console.WriteLine($"Subtraction of given inputs : {obj.Subtract(input1, input2)}");
+            Console.WriteLine($"Multiplication of given inputs : {obj.Multiply(input1, input2)}");
+            Console.WriteLine($"Division of given inputs : {obj.Divide(input1, input2)}");
+        }
+    }
+
+    /// <summary>
+    /// This the given class where all the functions are implemented
+    /// </summary>
+    public class MathUtils
+    {
+        /// <summary>
+        /// This function will give addition of two numbers
+        /// </summary>
+        /// <param name="input1"> First input element</param>
+        /// <param name="input2"> Second input element</param>
+        /// <returns>This function will return integer</returns>
```

Comment:
```suggestion
        /// <returns>Returns sum of input1 and input2</returns>
```
Change similarly for other methods as well